pkg/proto/blobstore/blobstore.proto

```diff
@@ -73,8 +73,13 @@ message CircularBlobAccessConfiguration {
 }
 
 message GRPCBlobAccessConfiguration {
-    // Endpoint address of the GRPC server (e.g., "localhost:8982").
+    // Endpoint address of the GRPC server (e.g., "http://localhost:8982").
```

Comment:
Let's move TLS client details to a specific message then (holding `server_ca_file`, `client_cert_file` and `client_key_file`). We'll need an explicit boolean somewhere to activate TLS if we are not deducing it from the URL in order to support client-unauthenticated TLS encrypted connection to trusted server though.

Reply:
> We'll need an explicit boolean somewhere to activate TLS if we are not deducing it from the URL in order to support client-unauthenticated TLS encrypted connection to trusted server though.

In that case we could use the empty message for that, right? Protobuf makes a difference between an empty message and one that is absent. For Golang, you see that nested messages use pointers. When no nested message is provided, the pointer is nil. From a configuration file point of view, it will look like this:

```
// No TLS.
grpc {
  endpoint: "localhost:8982"
}

// TLS with default options.
grpc {
  endpoint: "localhost:8982"
  tls {}
}

// TLS with custom CA, etc.
grpc {
  endpoint: "localhost:8982"
  tls {
    server_ca: "..."
    ...
  }
}
```

Reply:
Perfect!